content/components/core/code/scss-variables.md

```diff
@@ -0,0 +1,71 @@
+| `$AU-color-foreground-action` | `#00698F`       | Colour of links, buttons or other action items.                                                             |
+| `$AU-color-foreground-focus`  | `#9263DE`       | Color around an element that is in focus state. |
+| `$AU-color-background`        | `#FFFFFF`       | Background colour.                                                                            |
+
+
+### Dark Theme Colours
+
+| Variable Name               | Default Value | Description                                                                                                |
+|-----------------------------|---------------|------------------------------------------------------------------------------------------------------------|
+| `$AU-colordark-foreground-text`   | `#FFFFFF`       | Colour used for paragraphs, headings etc.                                                                                  |
+| `$AU-colordark-foreground-action` | `#61DAFF`       | Colour of links, buttons or other action items.                                                             |
+| `$AU-colordark-foreground-focus`  | `#C390F9`       | Color around an element that is in focus state. |
+| `$AU-colordark-background`        | `#135E70`       | Background colour.                                                                            |
+
+
+### Fonts
+
+| Variable Name      |  Default Value                                                                                                                         |  Description                                                          | 
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------| 
+| `$AU-font`           |  `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"` |  Fonts used for body text.                                                       | 
+| `$AU-font-monospace` |  `"SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace`                                                              |  Fonts used for mathematical and code related HTML elements | 
+| `$AU-fontsize-map`   |  ( xs: 14, sm: 16, md: 20, lg: 24, xl: 32, xxl: 40, xxxl: 48 ) |  Pixel sizes from a 1.25 typescale rounded to the nearest `$AU-unit` (vertical grid, default value at 4) | 
+| `$AU-lineheight-map` |  ( nospace: 1, heading: 1.25, default: 1.5 )                   |  Lineheight mapped to keyword                                                      | 
+
+
+### Media Breakpoints
+
+Our breakpoints are designed for mobile first. These breakpoints correspond to `xs`, `sm`, `md` and `lg` in the Design System [media query function](#au-media).
```

Comment:
I feel like the second sentance takes it time to explain what you are saying, maybe try:

These breakpoints are used in the `AU-media` function and are referenced by `xs`...